setup.py

```diff
@@ -26,6 +26,7 @@
     'recommonmark>=0.4.0',
     'Sphinx>=1.3.5',
     'sphinxcontrib-napoleon>=0.4.4',
+    'sphinx-rtd-theme>=0.1.9',
```

Comment:
Oh, interesting. In that case, we should delete `bigchaindb/docs/requirements.txt` and modify the instructions in [`bigchaindb/docs/README.md`](https://github.com/bigchaindb/bigchaindb/blob/develop/docs/README.md)